### PR TITLE
hits and lin regr doc updates

### PR DESCRIPTION
**Pull request by fmcquillan99**
minor updates to graph HITS algo docs
fixed order of params in lin regr prediction docs

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/287/

File: src/ports/postgres/modules/graph/hits.sql_in [hunk 1]
```diff
@@ -103,18 +102,18 @@ It is named by adding the suffix '_summary' to the 'out_table'
 parameter.
 
 <dt>max_iter (optional) </dt>
-<dd>INTEGER, default: 100. The maximum number of iterations allowed. An 
+<dd>INTEGER, default: 100. The maximum number of iterations allowed. One
```
Comment (rahiyer) [inline src/ports/postgres/modules/graph/hits.sql_in]: s/one/each

Comment (iyerr3) [inline src/ports/postgres/modules/graph/hits.sql_in]: s/One/Each

File: src/ports/postgres/modules/graph/hits.sql_in [hunk 2]
```diff
@@ -103,18 +102,18 @@ It is named by adding the suffix '_summary' to the 'out_table'
 parameter.
 
 <dt>max_iter (optional) </dt>
-<dd>INTEGER, default: 100. The maximum number of iterations allowed. An 
+<dd>INTEGER, default: 100. The maximum number of iterations allowed. One 
     iteration consists of both authority and hub phases.</dd>
 
 <dt>threshold (optional) </dt>
 <dd>FLOAT8, default: (1/number of vertices * 1000).
+    Threshold must be set to a value between 0 and 1.
```
Comment (iyerr3) [inline src/ports/postgres/modules/graph/hits.sql_in]: Is this inclusive of end points?

File: src/ports/postgres/modules/regress/linear.sql_in [hunk 3]
```diff
@@ -183,16 +183,15 @@ FROM (
 @par Prediction Function
 The prediction function is as follows:
 <pre class="syntax">
-linregr_predict(coef, col_ind)
+linregr_predict(col_ind, coef)
```
Comment (iyerr3) [inline src/ports/postgres/modules/regress/linear.sql_in]: This is inconsistent with the actual function, which has `coef` as the first parameter. I believe the examples incorrectly place the `col_ind` first.

Comment (fmcquillan99) [reply src/ports/postgres/modules/regress/linear.sql_in]: Yes, you are correct.  From
http://madlib.apache.org/docs/latest/linear_8sql__in.html

float8 linregr_predict | ( float8 [] | coef, float8 [] | col_ind_var)

I will revert and correct the examples

File: src/ports/postgres/modules/graph/hits.sql_in (hunk 2 above)
Comment (fmcquillan99) [reply src/ports/postgres/modules/graph/hits.sql_in]: yes, range is [0,1]
0 means run full num_iter specified

Will clarify in docs

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/288/

**Comment by iyerr3** [comment]
Commit daf67f81b merges this PR. This can be closed.

**Pull request closed without merging**